src/components/tagApplyMenu/AposTagApply.vue

```diff
@@ -39,7 +46,9 @@
               :disable-focus="!open"
             />
           </p>
-          <span class="apos-apply-tag-menu__empty-icon">🌾</span>
+          <span class="apos-apply-tag-menu__empty-icon">
+            🌾
```

Comment:
? No lie, at first I thought this was a smudge on my screen...

Reply:
lol